### PR TITLE
[Python] querying articles batch 3 [Replace C# samples]

### PR DESCRIPTION
**Pull request by reebhub**
Related issues:
[RDoc-2701](https://issues.hibernatingrhinos.com/issue/RDoc-2701/Python-Client-API-Session-Querying-Filter-by-non-existing-key-Replace-C-samples) - Filter by non-existing key
[RDoc-2700](https://issues.hibernatingrhinos.com/issue/RDoc-2700/Python-Client-API-Session-Querying-Filter-by-field-presence-Replace-C-samples) - Filter by field presence
[RDoc-2704](https://issues.hibernatingrhinos.com/issue/RDoc-2704/Python-Client-API-Session-Querying-More-like-this-Replace-C-samples) - More like this
[RDoc-2702](https://issues.hibernatingrhinos.com/issue/RDoc-2702/Python-Client-API-Session-Querying-Using-intersect-Replace-C-samples) - Using intersect
[RDoc-2703](https://issues.hibernatingrhinos.com/issue/RDoc-2703/Python-Client-API-Session-Querying-Perform-lazy-query-Replace-C-samples) - Perform lazy query
[RDoc-2709](https://issues.hibernatingrhinos.com/issue/RDoc-2709/Python-Client-API-Session-Querying-Query-for-suggestions-Replace-C-samples) - Query for suggestions

### CODE REVIEW

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-filter-by-field.python.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+from Employees
+where exists("FirstName")
+{CODE-TAB-BLOCK/}
+{CODE-TABS/}
+
+{PANEL/}
+
+{PANEL: Filter by field path }
+
+To search for documents that contain a specific field by the field **path**, pass the path to `where_equals`.
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-filter-by-field.python.markdown:40]:
```suggestion
To search for documents that contain a specific field by the field **path**, pass the path to `where_exists`.
```

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-filter-by-non-existing-field.python.markdown [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+----
+
+{NOTE/}
+
+---
+
+{PANEL: Query the collection (dynamic query)}
+
+To run a dynamic query over a collection and find which documents are missing a specified field,  
+use the `not_` and `where_exists` extension methods, accessible from the [document_query](../../../client-api/session/querying/document-query/what-is-document-query) API,
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-filter-by-non-existing-field.python.markdown:23]: Extension methods are unavailable in Python
```suggestion
use the `not_` and `where_exists` methods, accessible from the [document_query](../../../client-api/session/querying/document-query/what-is-document-query) API, 
```

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-use-morelikethis.python.markdown [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# Using MoreLikeThis
+
+`more_like_this` is available through query extension methods, and will return similar documents according
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-use-morelikethis.python.markdown:3]:
```suggestion
`more_like_this` is available through query methods, and will return similar documents according 

```

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-work-with-suggestions.python.markdown [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+{PANEL: Syntax}
+
+**Suggest using**:
+
+{CODE:python syntax_1@ClientApi\Session\Querying\HowToWorkWithSuggestions.py /}
+
+| Parameter      | Type                                         | Description  |
+|----------------|----------------------------------------------|--------------|
+| **suggestion_or_builder** (Union) | `SuggestionBase`<br>-or-<br>`Callable[[SuggestionBuilder[_T]], None]` | Suggestion instance<br>-or-<br>Suggestion builder |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-work-with-suggestions.python.markdown:176]: lets enhance it

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-work-with-suggestions.python.markdown [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+
+| Return type    | Description  |
+|----------------|--------------|
+| `SuggestionDocumentQuery[_T]` | Retrieved suggestions |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-work-with-suggestions.python.markdown:180]: it's the query, not results - you can do something more with it or call `execute()` on it

**Review by poissoncorp — CHANGES_REQUESTED** [review]
lets enhance it a bit

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-filter-by-field.python.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-filter-by-field.python.markdown:40]: name and path sections united

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-filter-by-non-existing-field.python.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-filter-by-non-existing-field.python.markdown:23]: done

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-use-morelikethis.python.markdown (hunk 3 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-use-morelikethis.python.markdown:3]: done

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-work-with-suggestions.python.markdown (hunk 4 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-work-with-suggestions.python.markdown:176]: done

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-work-with-suggestions.python.markdown (hunk 5 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/how-to-work-with-suggestions.python.markdown:180]: done

**Review by poissoncorp — APPROVED** [review]

**Pull request merged**